pkgs/development/python-modules/yaramod/default.nix

```
@@ -0,0 +1,49 @@
+{ lib
+, buildPythonPackage
+, cmake
+, fetchFromGitHub
+, pytestCheckHook
+, libxcrypt
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "yaramod";
+  version = "3.12.2";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "avast";
+    repo = pname;
+    rev = "refs/tags/v${version}";
+    hash = "sha256-1q+lzNtm8qko9ZAxJjkmPOQjHD5GxB6YyEz0tr+QWGk=";
+  };
+
```

```suggestion

  postPatch = ''
    rm -r deps
  '';

```
We try not to use vendored dependencies.